packages/compass-aggregations/src/components/pipeline/collation-toolbar.module.less

```diff
@@ -29,22 +29,22 @@
   padding: 4px;
   margin-right: 5px;
 
-  .collation-toolbar-input-label {
+  .toolbar-input-label {
     background: @gray5;
     color: @pw;
     white-space: nowrap;
     border-radius: 14px;
-    padding: 2px 1px;
+    padding: 2px 4px;
```

Comment:
`padding-left: 4px` makes sense to me, but on the right this feels very cramped, maybe let's do `padding-right: 8px` here?